openwisp_controller/config/base/device.py

```diff
@@ -163,6 +163,28 @@ def generate_key(self, shared_secret):
         else:
             return KeyField.default_callable()
 
+    def _validate_org_device_limit(self):
+        if not self._state.adding:
+            # This check is only executed when a new device
+            # is created.
+            return
+        if (
+            not hasattr(self, 'organization')
+            or self.organization.config_limits.device_limit == 0
+        ):
+            return
+        registered_devices = self._meta.model.objects.filter(
+            organization=self.organization
+        ).count()
+        if registered_devices >= self.organization.config_limits.device_limit:
+            raise ValidationError(
+                _(
+                    ('Device registration limit reached for {org} organization').format(
```

Comment:
What do you think about the following?
```suggestion
                    ('The maximum amount of allowed devices has been reached for organization {org}.').format(
```